### PR TITLE
Add Star Trek: The Original Series

### PR DESCRIPTION
**Pull request by augustopedro**

### CODE REVIEW

**Comment by RichardLitt** [comment]
Hey, thanks @augustopedro! Could you add a comment about why this is awesome?

**Comment by augustopedro** [comment]
Yeah, of course! I'll add the rest of the Star Trek spin-offs later.

File: readme.md [hunk 1]
```diff
@@ -1328,6 +1328,12 @@ The show feels completely different than Stargate SG-1 and Atlantis, it is certa
 
 > Trapped on an Ancient spaceship billions of light years from home, a group of soldiers and civilians struggle to survive and find their way back to Earth.
 
+#### [Star Trek: The Original Series](http://www.imdb.com/title/tt0060028/) [8.4]
+
+A science-fiction masterpiece, Star Trek has been a really important vision not only of what future technology could look like, but also a reflection of what the hopes were, especially in his time (The 60's), for what human society could look like. - [@augustopedro](https://github.com/augustopedro)
```
Comment (sindresorhus) [inline readme.md]: `especially in his time (The 60's)` => `especially in the 60's`

**Comment by sindresorhus** [comment]
Question. I tried watching it some time back, but found it slow and boring so I gave up. Does it get better after a while or is it just not for me?

**Comment by parro-it** [comment]
You have to had watched it at the age of 10 and than you'll find it awesome forever :smiley:

**Comment by augustopedro** [comment]
The Original Series, in many ways, is Star Trek at its purest. It's true that it could be campy and slow, but don't mistake that for it being unsophisticated. It was very smartly written, and while it can vary wildly in quality (like many shows from the 60's) it has a solid philosophical core, and often had some interesting things to say.

Plus, I'm not sure the franchise has ever quite topped the amazing chemistry between Shatner, Nimoy, and Kelly. It's worth watching for them alone.

**Comment by sindresorhus** [comment]
@augustopedro Thanks, I guess I'll try again with a more open mind! Your above comment is great, can you add that too to the readme?

**Comment by augustopedro** [comment]
Ok! Added!

**Comment by sindresorhus** [comment]
Perfect! :)

**Pull request closed without merging**